conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -12,6 +12,7 @@ jobs:
 {%- endif %}
 {%- if azure.free_disk_space %}
   - script: |
+{%- if (azure.free_disk_space == true) or ('cache' in azure.free_disk_space) %}
```

Comment:
This will break if we had `azure: free_disk_space: false` right?

Reply:
We check truthiness on line 13 before getting to this line